airbyte-workers/src/main/java/io/airbyte/workers/temporal/check/connection/CheckConnectionActivityImpl.java

```diff
@@ -55,9 +55,9 @@ public CheckConnectionActivityImpl(final WorkerConfigs workerConfigs,
     this.airbyteVersion = airbyteVersion;
   }
 
-  public StandardCheckConnectionOutput run(final JobRunConfig jobRunConfig,
-                                           final IntegrationLauncherConfig launcherConfig,
-                                           final StandardCheckConnectionInput connectionConfiguration) {
+  public StandardCheckConnectionOutput check(final JobRunConfig jobRunConfig,
```

Comment:
this rename was necessary, otherwise we were trying to register two activities whose entrypoint was called `run` which confused temporal